src/components/primitive/navbar/navbar.component.props.ts

```diff
@@ -4,7 +4,7 @@ import { TextComponentProps } from './../text/text.component.props';
 /**
  * Different screen sizes used across the screens.
  */
-export enum ScreenSize {
+export enum BreakPoints {
```

Comment:
This enum seems to be duplicated. This should be moved to a common folder like Theme/Utils.